config/kibana.yml

```diff
@@ -61,6 +61,11 @@
 # must be a positive integer.
 # elasticsearch.requestTimeout: 30000
 
+# List of Kibana client-side headers to send to Elasticsearch. To send *no* client-side
+# headers, set this value to null, not []. Setting it to [] will send the default list
```

Comment:
Agreed re: solving this programmatically rather than passing on the burden to the user.

The issue isn't so much that Joi doesn't allow empty arrays — it does. Its more that we want it to see an empty array and treat it as-is, rather than replacing it with our specified default (which would be `[ 'authorization' ]`.